test/distributed/org/apache/cassandra/distributed/test/NodeToolTest.java

```diff
@@ -67,9 +72,37 @@ public void testCaptureConsoleOutput() throws Throwable
     public void testNodetoolSystemExit()
     {
         // Verify currently calls System.exit, this test uses that knowlege to test System.exit behavior in jvm-dtest
-        CLUSTER.get(1).nodetoolResult("verify", "--check-tokens")
-               .asserts()
-               .failure()
-               .stdoutContains("Token verification requires --extended-verify");
+        NODE.nodetoolResult("verify", "--check-tokens")
```

Comment:
We have been trying to consolidate tooling testing across the board with a recently introduced `ToolRunner`. I would move these new tests to that new interface for consistency and to benefit from any future dev there. See `NodeToolRingTest` for some example

Reply:
for jvm-dtest I would not agree with this.  `NodeToolResult` is more fleshed out than tool runner offers (as it is deeply integrated with jvm-dtest), and the added ToolRunner wrapper adds more complexity without offering simplification to test authors.

I feel that ToolRunner should be used when calling `main` or calling a CLI on disk, but not when using jvm-dtest nodetool.

Reply:
I align with David on the usage. 
The `ToolRunner` is good for non-distributed cases. Meanwhile, the `nodetool` and `NodeToolResult` interfaces exposed from jvm-dtest-api is suitable in the jvm dtest cases. 
The nodetool api is fluent and easy to use (e.g. assertion). The `NodeToolResult` already contains stdout and stderr from command. Therefore I do not see benefits from wrapping the jvm-dtest-api within ToolRunner interface. We lose the benefits by wrapping the api into `ToolRunner`.